File: clientJava/controllerApi/src/main/java/org/evomaster/clientJava/controllerApi/dto/SutRunDto.java
package org.evomaster.clientJava.controllerApi.dto;

public class SutRunDto {

    /**
     * Whether the SUT should be running
     */
    public Boolean run;

    /**
     * Whether the internal state of the SUT should be reset
     */
    public Boolean resetState;


    public SutRunDto() {
    }

    public SutRunDto(Boolean run, Boolean resetState) {
        this.run = run;
        this.resetState = resetState;
    }
}

File: client-js/src/controller/api/dto/SutRunDto.ts

export default class SutRunDto {

    /**
     * Whether the SUT should be running
     */
    run: boolean;

    /**
     * Whether the internal state of the SUT should be reset
     */
    resetState: boolean;

    /**
     *  Whether SQL heuristics should be computed.
     *  Note: those can be very expensive
     */
    calculateSqlHeuristics: boolean;

    /**
     *  Whether SQL execution info should be saved.
     */
    extractSqlExecutionInfo: boolean;

}
